Objects/unicodeobject.c

```diff
@@ -9381,6 +9381,7 @@ _PyUnicode_InsertThousandsGrouping(
     PyObject *thousands_sep,
     Py_UCS4 *maxchar)
 {
+    min_width = Py_MAX(0, min_width);
```

Comment:
I would prefer to fix the caller of this function instead: Python/formatter_unicode.c.

Reply:
Thanks Victor. Few clarifications : 

* By fixing the caller do you mean `min_width` as in `min_width = Py_MAX(0, spec->n_min_width);` and passing it to `_PyUnicode_InsertThousandsGrouping` from Python/formatter_unicode.c without modifying spec->n_min_width ? There is an explicit comment that `spec->n_min_width` and I don't want to mutate the `spec->n_min_width` . 

* `_PyUnicode_InsertThousandsGrouping` is used in two places in formatter_unicode.c and does both places need this change ?

Reply:
Nevermind, I changed my mind and "min_width = Py_MAX(0, min_width);" in _PyUnicode_InsertThousandsGrouping() is just fine.